src/app/modules/dashboard/pages/dashboard/components/cpu-stats/cpu-stats.component.html

```diff
@@ -0,0 +1,14 @@
+<mat-card>
+    <mat-card-content>
+        <h2>CPU</h2>
+        <hr style="border-color: #2196F3;">
+        <canvas baseChart width="350" height="200" style="margin-top: 1em;"
```

Comment:
There should be a spinner here. Also, lowering the header to `h3` would make it look a bit better.
```suggestion
        <h3>CPU</h3>
        <div class="blue-border">&nbsp;</div>
        <mat-progress-spinner mode="indeterminate" strokeWidth="2" *ngIf="loading$ | async"></mat-progress-spinner>
        <canvas baseChart width="350" height="200" [style.display]="(loading$ | async) ? 'none' : ''" style="margin-top: 1em;"
```
Please apply this change to the other components as well (jvm memory / threads)